webpack.config.js

```diff
@@ -145,6 +145,12 @@ export default (env, args) => {
       // NEUROGLANCER_SHOW_OBJECT_SELECTION_TOOLTIP: true
 
       // NEUROGLANCER_GOOGLE_TAG_MANAGER: JSON.stringify('GTM-XXXXXX'),
+      /**
+       * To deploy to a different origin, you will need to generate your
+       * own Client ID from Globus and substitute it in.
+       * @see https://docs.globus.org/api/auth/developer-guide/#developing-apps
+       */
+      GLOBUS_CLIENT_ID: JSON.stringify("f3c5dd86-8c8e-4393-8f46-3bfa32bfcd73"),
```

Comment:
This default value is a Client ID that is managed by the Globus team. 

It is something that _could_ be distributed as part of the codebase or removed as a default and only referenced here as a comment.

Reply:
:+1: from my side for at least not needing a fork to set a different value.

Reply:
@joshmoore – that was my intent here, but I am a bit unsure this method works as intended...

I _thought_ something like `npm run build -- --define GLOBUS_CLIENT_ID=example` was the intended use, but [it looks like the `NEUROGLANCER_CLI` environment flag needs to be disabled](https://github.com/google/neuroglancer/blob/master/webpack.config.js#L150) in order for the incoming `--define` properties to be merged. 

Running `npm run build -- --env NEUROGLANCER_CLI=false ...` encounters an error due to the [`.strict()` usage in `build_tools/cli.ts`](https://github.com/google/neuroglancer/blob/master/build_tools/cli.ts#L262) – I don't want to derail the addition of this functionality, but just wanted to make sure I had a better grasp on the change and how it is used.